app/config.py

```diff
@@ -562,6 +562,7 @@ class Config(object):
     FF_CELERY_CUSTOM_TASK_PARAMS = env.bool("FF_CELERY_CUSTOM_TASK_PARAMS", True)
     FF_CLOUDWATCH_METRICS_ENABLED = env.bool("FF_CLOUDWATCH_METRICS_ENABLED", False)
     FF_SALESFORCE_CONTACT = env.bool("FF_SALESFORCE_CONTACT", False)
+    FF_NEW_BRANDING = env.bool("FF_NEW_BRANDING", False)
```

Comment:
Wrong FF! Should be `FF_TEMPLATE_CATEGORY`!  We'll also need a manifests release to make sure this FF is being passed to API, as I don't think it currently is.